packages/simulator/__tests__/unit/car.js

```diff
@@ -124,58 +158,52 @@ describe('A car', () => {
     expect(car.queue).toHaveLength(10)
   })
 
-  it('should be able to handle the bookings from the same place in the queue', function (done) {
+  it('should be able to handle the bookings from the same place in the queue', async () => {
```

Comment:
This test now works in isolation, but not together with the others. We are leaking between tests somehow. Not figured that out yet.